test/powershell/engine/Jobs.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe 'Basic Job Tests' -Tags 'CI' {
+
+    $job = Start-Job {1}
+
+    It 'Test job creation' {
+        $job | should not be $null
+    }
+
+    It 'Test job State' {
+        Wait-Job $job -Timeout 5
+        $job.JobStateInfo.State -eq 'Completed' | should be $true
+    }
+
+    It 'Job output test' {
+        Receive-Job $job -wait | should be 1
+    }
+}
```

Comment:
Is there a policy about missing `EOL` at `EOF`?

Personally, as someone who uses **hg** and **git** (and **Linux**), missing `EOL` markers at `EOF` result in lots of pain which I'd rather avoid.